ffi/diplomat/Cargo.toml

```diff
@@ -71,8 +71,8 @@ tinystr = { version = "0.4.10", features = ["alloc"], default-features = false }
 writeable = { path = "../../utils/writeable/" }
 
 # Run `cargo make diplomat-install` to get the right diplomat binary installed
-diplomat = { git = "https://github.com/rust-diplomat/diplomat", rev = "4d8450f575a7839e3bc2d48aedae5408f1825c30" }
-diplomat-runtime = { git = "https://github.com/rust-diplomat/diplomat", rev = "4d8450f575a7839e3bc2d48aedae5408f1825c30" }
+diplomat = { git = "https://github.com/rust-diplomat/diplomat", rev = "# This file is automatically @generated by Cargo." }
+diplomat-runtime = { git = "https://github.com/rust-diplomat/diplomat", rev = "" }
```

Comment:
Is this intentional? This is the source of truth for the version number, right?

Reply:
oops!